Monika After Story/game/script-topics.rpy

```diff
@@ -9509,6 +9509,66 @@ label monika_bullying:
 
     return "derandom"
 
+init 5 python:
+    addEvent(Event(persistent.event_database,eventlabel="monika_song_yozurina",category=['songs'],prompt="Yozurina",pool=True, random=True))
```

Comment:
This will need to be moved to the new script file. You can use the `aff_range` property to handle the affection restriction on this.

This will also need a shorter variant of this song to unlock the full one in the pool delegate.